src/components/stegvisning/stegvisning-hooks.tsx

```diff
@@ -24,8 +34,61 @@ export const useStegvisningNavigation = (currentPath: Path) => {
         : stegvisningOrder
     navigate(stepArrays[stepArrays.indexOf(currentPath) + 1])
   }
+
   const onStegvisningPrevious = () => {
-    navigate(-1)
+    let antallStepTilbake = 1
+    const isEndring = loependeVedtak && isLoependeVedtakEndring(loependeVedtak)
+    const stepArrays = isEndring ? stegvisningOrderEndring : stegvisningOrder
+
+    const currentPathIndex = stepArrays.indexOf(currentPath)
+
+    // Hvis brukeren er på eller forbi afp steget
+    if (
+      (!isEndring && currentPathIndex >= 4) ||
+      (isEndring && currentPathIndex >= 2)
+    ) {
+      // Bruker med uføretrygd som er eldre enn AFP-Uføre oppsigelsesalder har ikke fått steg om AFP og skal navigere tilbake forbi den
+      if (
+        ufoeregrad &&
+        foedselsdato &&
+        isFoedselsdatoOverEllerLikMinUttaksalder(foedselsdato)
+      ) {
+        antallStepTilbake = antallStepTilbake + 1
```

Comment:
Kan eventuelt skrives slik:
```suggestion
        antallStepTilbake++
```

Reply:
Vi har ikke skrudd det på her, men jeg har fått med meg fra andre prosjekter at eslint "no-plusplus" var en grei regel å følge. Skriver derfor iterasjoner med +1.

Reply:
Sant det! Ser at man også kan skrive `foo += 1` forresten (bare til info, trenger ikke endre).